From 56adfa74711d029c0147ac2b3c1bb5e5ff6c2ba3 Mon Sep 17 00:00:00 2001
Message-Id: <56adfa74711d029c0147ac2b3c1bb5e5ff6c2ba3.1441918786.git.yunwei.zhang@intel.com>
In-Reply-To: <8631704be06e00cb7a7a415f13811ca4ece99e8b.1441918786.git.yunwei.zhang@intel.com>
References: <8631704be06e00cb7a7a415f13811ca4ece99e8b.1441918786.git.yunwei.zhang@intel.com>
From: Shubhangi Shrivastava <shubhangi.shrivastava@intel.com>
Date: Thu, 10 Sep 2015 13:50:14 +0530
Subject: [PATCH 10/10] REVERTME [VPG]: drm/i915: Fix DP audio registers

This patch fixes the offset used for DP lpe audio registers.

REVERTME: LP audio is not available upstream and may not be for
some more time which is required for this.

Change-Id: I118d10eb93e73f8724f855eb93c2aab9c2300dc6
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-10172
Signed-off-by: Sivakumar Thulasimani <sivakumar.thulasimani@intel.com>
Signed-off-by: Shubhangi Shrivastava <shubhangi.shrivastava@intel.com>
---
 drivers/gpu/drm/i915/i915_reg.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_reg.h b/drivers/gpu/drm/i915/i915_reg.h
index f487119..5c100f4 100644
--- a/drivers/gpu/drm/i915/i915_reg.h
+++ b/drivers/gpu/drm/i915/i915_reg.h
@@ -3016,8 +3016,8 @@ enum punit_power_well {
 #define AMP_UNMUTE					(1 << 1)
 #define AUD_CHICKEN_BIT_REG			0x62F38
 #define AUD_PORT_EN_B_DBG			0x62F20
-#define AUD_PORT_EN_C_DBG			0x62F28
-#define AUD_PORT_EN_D_DBG			0x62F2C
+#define AUD_PORT_EN_C_DBG			0x62F30
+#define AUD_PORT_EN_D_DBG			0x62F34
 #define VLV_AUD_CHICKEN_BIT_REG		(VLV_DISPLAY_BASE + AUD_CHICKEN_BIT_REG)
 #define VLV_AUD_PORT_EN_B_DBG		(VLV_DISPLAY_BASE + AUD_PORT_EN_B_DBG)
 #define VLV_AUD_PORT_EN_C_DBG		(VLV_DISPLAY_BASE + AUD_PORT_EN_C_DBG)
-- 
1.9.1

